releng/build.Jenkinsfile

```diff
@@ -7,7 +7,7 @@ pipeline {
   }
   tools {
     maven 'apache-maven-latest'
-    jdk 'oracle-jdk8-latest'
+    jdk 'openjdk-jdk11-latest'
```

Comment:
openjdk latest is not a good idea, it is not kept up to date. Use adopt instead.

Reply:
You mean temurin? :)

Reply:
yes `temurin-jdk11-latest`